src/main/java/com/mercadolibre/bootcampmelifrescos/exceptions/ApiException.java

```diff
@@ -0,0 +1,56 @@
+package com.mercadolibre.bootcampmelifrescos.exceptions;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.http.HttpStatus;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+@Getter
+@Setter
+public class ApiException extends Exception{
```

Comment:
```suggestion
public class ApiException extends Exception {
```

Adiciona um espaço ali, só pra fins de padronização mesmo.